### ISSUE TITLE
Dead links in lore.xhtml

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3484]
|<img alt="kehander1's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| kehander1 reported|
|-|-|
|Trac ID|trac#3484|
|Type|defect|
|Created|2008-10-21 21:16:37Z|

The homepage for MagicPoint has moved, and http://prosper.sourceforge.net is rather dead.

Attachments:

* [lorelinks.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/1d2/1d22545e9f5530b0432289b7766057eff2518f7d/387f39c98ccc4734ea9b4d59fd64a3f17dd69262.diff) (1166 bytes) - added by kehander1 on 2008-10-21 21:17:02Z - Fix for dead links in lore.xhtml

<details><summary>Searchable metadata</summary>

```
trac-id__3484 3484
type__defect defect
reporter__kehander1 kehander1
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__lore lore
keywords__documentation documentation
time__1224623797000000 1224623797000000
changetime__1226264070000000 1226264070000000
version__None None
owner__None None
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="spiv's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|spiv set status to `assigned`|
|-|-|

The fixes look good to me.  I'll merge them sometime soon.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

Thanks kehander1, merging it, seems spiv forgot about it.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [25332]) Apply lorelinks.diff: Fixing dead links in lore documentation.

Author: kehander1
Reviewer: spiv, thijs
Fixes: [#3484](https://github.com/twisted/twisted/issues/3484)
